nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -317,6 +350,8 @@ private static LibraryModels loadLibraryModels() {
 
     private final ImmutableSet<MethodRef> nullable_returns;
 
+    private final ImmutableSet<MethodRef> nonnull_returns;
```

Comment:
Camel case for this field and the previous one?

Reply:
Done. All other fields in this class too.